wiris/moodle-atto_wiris
# Fixed "Equations are not saved, when giving feedback"

This change fixes bug KB-5540 '[Moodle] Equations are not saved,
when giving feedback' (#69).

No changes have been made to the code, therefore a new behat test
has been added to make sure it works in all Moodle versions.

**How to reproduce**

- Edit an assigment
- Provide feedback
 -Add a formula using the atto_editor's MathType button
- Click on the feedback button
- Accept changes on the modal window that appears

**Expected result**

The formula is added in both comment editor's target element on the sidebar and the assignment page on the main side.

**Actual result**

The formula dissapears.
